save-cloud-common/src/commonMain/kotlin/org/cqfn/save/agent/Heartbeat.kt

```diff
@@ -27,11 +29,16 @@ data class ExecutionProgress(val percentCompletion: Int) {
  * @property agentId unique ID of the agent which sent the heartbeat
  * @property state current state of the Agent
  * @property executionProgress current progress of tests execution with this Agent
+ * @property timestamp the time of heartbeat posting
  */
 @Serializable
-data class Heartbeat(val agentId: String,
-                     val state: AgentState,
-                     val executionProgress: ExecutionProgress)
+data class Heartbeat(
+    val agentId: String,
+    val state: AgentState,
+    val executionProgress: ExecutionProgress,
+    @Contextual
+    val timestamp: LocalDateTime,
```

Comment:
Our `org.cqfn.save.utils.LocalDateTime` was intended as a workaround for Hibernate, which can store Java's LocalDateTime, but can't with kotlinx one. Here we don't depend on Hibernate, so we can use `kotlinx.datetime` normally, without workarounds with type aliases and custom serialization strategies.

Reply:
Changed to `kotlinx.datetime`